tests/end2endtest/test_server.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2021 IBM Corp. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Basic server tests.
+"""
+
+from __future__ import absolute_import, print_function
+
+import re
+
+# pylint: disable=unused-import
+from .utils import server_url  # noqa: F401
+# pylint: enable=unused-import
+from ..unit.utils import execute_pywbemcli
+
+
+def test_server_is_pegasus(server_url):
+    # pylint: disable=redefined-outer-name
+    """
+    Test that this WBEM server is OpenPegasus and that it provides the
+    expected Interop namespace, and the namespaces and profiles we want to test
+    against.
```

Comment:
Add this functionality is provided by the OpenPegasus  make build, make repository, make test repository including the building of repositories, providers, and namespaces. It is also based on OpenPegasus use of the DMTF schema version 2.4.1

Reply:
CIM Schema version 2.4.1 is certainly not correct. I checked all namespaces and the latest Version qualifier says 2.39.0.

For now, I have added the following paragraph:

```
    This container-packaged version of OpenPegasus is provided by the
    OpenPegasus project on GitHub (https://github.com/OpenPegasus/OpenPegasus).
    It includes OpenPegasus 14.2 (version string 2.14.2), the CIM Schema
    version 2.39.0, and a number of providers including a namespace provider.
```

Reply:
Karl has mixup in repo naming.  Clarify.

Reply:
DONE - adjusted proposed text to use Pegasus version 2.14.2, and CIM Schema 2.41.0.